fix: Follow 엔티티의 프로퍼티에 붙은 어노테이션 수정

rugii913 opened the pull request:
## 연관 이슈
- closes #8 

## 해당 작업을 한 동기/이유는 무엇인가요?
- Follow 엔티티의 @ Column 어노테이션 때문에 서버 실행이 안 됨

## 리뷰어에게 작업 또는 변경 내용을 상세히 설명해주세요
- [x] Follow 엔티티 val toUser, val fromUser에 붙어있는 @ Column 어노테이션을 @ JoinColumn으로 수정
    - 연관관계가 있는 객체 프로퍼티에 대해서는 @ Column이 아니라 @ JoinColumn을 써야한다.

rugii913 committed:
fix: Follow 엔티티의 프로퍼티에 붙은 어노테이션 수정

- Follow 엔티티 val toUser, val fromUser에 붙어있는 @Column 어노테이션을 @JoinColumn으로 수정
  - 연관관계가 있는 객체 프로퍼티에 대해서는 @Column이 아니라 @JoinColumn을 써야한다.

adorecamus commented on src/main/kotlin/com/dooingle/domain/follow/model/Follow.kt:
```diff
@@ -7,11 +7,11 @@ import jakarta.persistence.*
 class Follow(
     @ManyToOne
-    @Column
+    @JoinColumn
```
조인 컬럼 이름 to_user_id 로 하는 건 어떨까용...?ㅎㅎ

adorecamus commented on src/main/kotlin/com/dooingle/domain/follow/model/Follow.kt:
```diff
@@ -7,11 +7,11 @@ import jakarta.persistence.*
 
     @ManyToOne
-    @Column
+    @JoinColumn
```
여긴 from_user_id 어떠신지ㅎㅎ..!

rugii913 replied:
변경했습니당~

adorecamus reviewed (approved):
고생하셨습니다!! 그런데 조인 컬럼 이름 지정하는 건 어떠신지..!ㅎㅎ

rugii913 committed:
fix: @ JoinColumn 속성 name 추가

- @ JoinColumn 속성 name 추가
  - 참고: https://velog.io/@hellojihyoung/Spring-JoinColumn의-name과-referencedColumnName
